3.analyze_data/nbconverted/analyze_predicted_probabilities.py

```diff
@@ -138,26 +167,29 @@ def perform_dunn_median(_dmso_probs, _treatment_probs):
 
 # ## Defining tests and aggregation metric names
 
-# In[7]:
+# In[8]:
 
 
-comp_functions = {"dunn_test":
+comp_functions = {"dunn":
```

Comment:
consider adding a comment here to explain  the dict

Reply:
Added!